src/app/static/src/js/components/runWorkflow/workflowSummary/reportParameter.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+    <div id="workflow-summary" class="mt-3">
+        <div class="row">
+            <div class="col-12">
+                <div class="summary">
+                    <div v-for="(report, index) in dependencies.reports" :key="index" class="single-workflow-summary-area">
+                        <div class="workflow-summary-report wow fadeInLeft">
```

Comment:
I thought we were going to do without the animation classes like fadeIn? Don't think 'wow' belongs here either?

Reply:
removed